src/defaultConfig.js

```diff
@@ -34,15 +35,15 @@ import theme from './themes';
  * @name Configuration
  */
 const config = {
-  /** theme , the CSS style sheet
-   *
+  /**
    * theme , the CSS style sheet
    *
-   *| Parameter | Description |Type | Required | Values|
-   *| --- | --- | --- | --- | --- |
-   *| Theme |Built in Themes| String | Optional | Values include, default, forest, dark, neutral, null|
+   * | Parameter | Description |Type | Required | Values|
+   * | --- | --- | --- | --- | --- |
+   * | theme | Built in Themes | string | Optional | 'default', 'forest', 'dark', 'neutral', 'null'|
+   *
+   * **Notes:** To disable any pre-defined mermaid theme, use "null".
```

Comment:
Please make sure this is `'null'`, not `null`, I tried to search related source code, but I didn find any.